problems/variables.js

```diff
@@ -43,6 +63,15 @@
 //  * Store Dee's grade on a test to a variable
 //  * Find the average grade of all students
 //  * Print out if Dee's grade is higher than the class average
+let Alice = 100
```

Comment:
Always make variable names lowercase unless all caps to be constant. Some styles const ALICE = , but generally better alice